frontend/src/pages/ContributionView.jsx

```
@@ -0,0 +1,286 @@
+                  <p className="transcript-text">
+                    {currentVideo?.transcript || 'No transcript available for this video.'}
+                  </p>
+                </div>
+              )}
+              {activeTab === 'Notes' && (
+                <div className="notes-content">
+                  <textarea 
+                    className="notes-textarea" 
+                    placeholder="Take notes here..."
+                  ></textarea>
+                </div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)